prime-router/docs/playbooks/how-to-update-LIVD-table.md

```diff
@@ -0,0 +1,31 @@
+# How To Update The LIVD Table
+
+---
+## Introduction
+
+---
+HHS maintains a list of all authorized covid test devices as an xlsx. ReportStream
+keeps a local csv version copy of that list. This local csv file needs to be updated
+once the new LIVD table comes out. 
+
+## Steps
+
+---
+1. Download LIVD table from https://www.cdc.gov/csels/dls/sars-cov-2-livd-codes.html
+2. Click LOINC Mapping tab
+3. Save file in CSV format
+4. Follow naming convention: LIVD-SARS-CoV-2-latest.csv
```

Comment:
Are we keeping the other files that have dates?

Reply:
This was a question that I asked @RickHawesUSDS and he told me to keep them for the record and in case we needed to go back to one of them.